### ISSUE TITLE
Error when extracting date cell from Dataframe

### ISSUE THREAD

**Issue by kennethz3** [dovpanda-dev/dovpanda#80]
# Brief Description
When trying to extract one cell using iloc, dovpanda runs into an error. Sometimes, despite the error, this suggestion is made:
The shape of the returned series from slicing is (1,) Which suggests you are interested in the value and not in a new series. Try instead: `series.iat[row, col] `

# System Information

- OS: Ubuntu 18.04
- Python: 3.7.5
- Jupyter Lab: 1.14

# Minimally Reproducible Code

```python
df = pd.DataFrame([pd.Timestamp('01-01-2019')])
df.iloc[0][0]
```

# Error Messages

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-25-dc9af601df62> in <module>
      1 df = pd.DataFrame([pd.Timestamp('01-01-2019')])
----> 2 df.iloc[0][0]

~/anaconda3/envs/x/lib/python3.7/site-packages/dovpanda/base.py in run(*args, **kwargs)
    161                 for post in posts:
    162                     if self.similar <= post.stop_nudge:
--> 163                         post.replacement(ret, arguments)
    164             return ret
    165 

~/anaconda3/envs/x/lib/python3.7/site-packages/dovpanda/core.py in suggest_at_iat(res, arguments)
    186 def suggest_at_iat(res, arguments):
    187     self = arguments.get('self')
--> 188     shp = res.shape
    189     if res.ndim < 1:  # Sometimes specific slicing will return value
    190         return

AttributeError: 'Timestamp' object has no attribute 'shape'
```

**Comment by DeanLa** [comment]
Thanks!
A similar error as been issues in #77, will be closed by #79 and released on next version.

**Comment by DeanLa** [comment]
some behind the scenes: The dovpanda hint jumps on the `df.iloc[0]` part, while the error is raised by the second `[0]`. A workaround until next release: try df.iat[0,0]
